style.css

```diff
@@ -11,603 +11,624 @@ body {
   background-color: #e5e5e5;
 }
 
-.header {
-  display: flex;
-  width: 100%;
+#head {
   height: 100vh;
-  padding: 0 200px;
-  border-radius: 0 0 0 100px;
+}
+
+#head nav {
+  display: flex;
+  justify-content: center;
   background-color: #fff;
 }
 
-.header nav {
+#head nav .nav-inner {
   display: flex;
   justify-content: space-between;
-  padding: 20px 150px 50px 150px;
+  padding: 30px 0 70px 0;
+  background-color: #fff;
+  width: 1440;
 }
 
-.content button a {
-  color: #396df2;
-  transition: all 1s;
+#head .toggle-button {
+  display: none;
 }
 
-.contact-container form button a {
-  transition: all 1s;
+#head #hamburger-menu {
+  display: flex;
 }
 
-.content .btn3 a {
-  color: #fff;
+.about-container .box1 ul li {
+  margin: 0 20px 20px 0;
 }
 
-.header nav .logo a {
-  font-weight: 700;
-  font-size: 18px;
-  line-height: 20px;
-  color: #6070ff;
+.about-container .box2 .about-list > li {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 40px;
 }
 
-.header .toggle-button {
-  display: none;
+.about-container .box2 .about-list li {
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  color: #344563;
 }
 
-.header nav #hamburger-menu {
-  display: flex;
+.about-container .box2 .icon-list li {
+  width: 122px;
+  height: 120px;
+  padding-top: 20px;
+  padding-left: 10px;
+  background: #f7f7f9;
+  border-radius: 8px;
+  margin-right: 10px;
 }
 
-.content button a:hover {
-  color: #fff;
+#head ul li {
+  padding-right: 20px;
 }
 
-.about-myself .about-content button a {
-  font-weight: 500;
-  font-size: 17px;
-  line-height: 24px;
-  color: #396df2;
-  transition: all 1s;
+#head #hamburger-menu li {
+  padding: 0;
 }
 
-.section-odd .content ul li a {
-  background-color: #ebebff;
-  padding: 5px 10px;
-  border-radius: 10px;
-  font-weight: 500;
-  font-size: 12px;
-  line-height: 16px;
-  color: #6070ff;
+#head #hamburger-menu li:nth-child(1) {
+  margin-right: 30px;
 }
 
-.section-even .content ul li a {
-  background-color: #ebebff;
-  padding: 5px 10px;
-  border-radius: 10px;
+#head #hamburger-menu li:nth-child(2) {
+  margin-right: 30px;
+}
+
+.card-content ul li a {
   font-weight: 500;
   font-size: 12px;
   line-height: 16px;
   color: #6070ff;
+  background-color: #ebebff;
+  padding: 5px 10px;
+  border-radius: 7px;
 }
 
-.contact-container form .btn button a {
+#head .logo a {
```

Comment:
- [ ] Kindly try to increase some padding of the inputs to be like the Figma Design😃

> Check below to see the difference 😊⤵️

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/214842209-96d3bcdf-f715-4edd-bac2-42d4723811bf.png)  |    ![image](https://user-images.githubusercontent.com/83360636/214842156-34ff2d0a-2ba6-47a1-a063-9b60894b5ad9.png) |

Reply:
Fixed.